Cesium.CodeGen/Cesium.CodeGen.csproj

```diff
@@ -8,6 +8,8 @@
 
     <ItemGroup>
       <PackageReference Include="Mono.Cecil" Version="0.11.4" />
+      <PackageReference Include="QuikGraph" Version="2.5.0" />
+      <PackageReference Include="QuikGraph.Graphviz" Version="2.5.0" />
```

Comment:
Not needed depedency

Reply:
I am not against it if you use it for some diagnostic purposes, but please make sure to leave at least some instructions on the usage. We can then implement an option to emit a graph, or whatever.